fixtures/plugins/drizzle/drizzle.config.ts

```diff
@@ -0,0 +1,5 @@
+import type { Config } from "drizzle-kit";
+
+export default {
+  schema: "./src/schema.ts",
```

Comment:
I think ideally we should mark this file as entrypoint as well, but not sure how to do that.

Reply:
Inside `findDependencies` you can `load` the `configFilePath` (first argument) and then return the value `config.schema` in the array of dependencies.

Reply:
Thanks, on it. Which lib should I use to expand/process globs?

Reply:
That should be mostly handled, the plugin can literally return e.g. `['.src/schema.ts']` and Knip will figure it out (e.g. abs/rel paths are entries, the rest is package names).

The first argument for `findDependencies` is `configFilePath` and that should be an absolute path already.

Reply:
Here's a similar example: https://github.com/webpro/knip/blob/main/src/plugins/commitizen/index.ts#L25-L26

Reply:
`drizzle` accepts globs as well, I tried returning them but it didn't work, so I now process the paths with globby before returning the array, it did the trick. 👍